drivers/sensor/ti_hdc20xx/ti_hdc20xx.c

```diff
@@ -24,6 +25,12 @@ LOG_MODULE_REGISTER(TI_HDC20XX, CONFIG_SENSOR_LOG_LEVEL);
 #define TI_HDC20XX_MANUFACTURER_ID	0x5449
 #define TI_HDC20XX_DEVICE_ID		0x07D0
 
+/* Register bits */
+#define TI_HDC20XX_BIT_CONFIG_SOFT_RES		0x80
+
+/* Reset time: not in the datasheet, but found by trial and error */
+#define TI_HDC20XX_RESET_TIME		K_MSEC(1)
```

Comment:
Was 1ms the absolute lowest time you could use, or does this include some wiggle room?
Given that this value is not from any datasheet, it would be handy to note the lowest duration that worked here as well.
If 1ms was the lowest duration, you should add some wiggle room here to account for chip variations

Reply:
In practice, I have found that a single I2C transaction at 400kHz is a big enough delay, so I took a lot of margin.